src/main/java/in/koreatech/koin/domain/timetableV2/dto/response/TimetableLectureResponse.java

```diff
@@ -83,7 +82,25 @@ public record ClassInfo(
             public static List<ClassInfo> of(String classTime, String classPlace) {
                 // 정규 강의인 경우 강의 장소가 없기 때문에 바로 반환
                 if (classPlace == null) {
-                    return List.of(new ClassInfo(parseClassTimes(classTime), null));
+                    String[] split = classTime.substring(1, classTime.length() - 1).trim().split(",\\s*");
```

Comment:
정규 강의인 경우 getClassTime 메소드를 사용해서 null 체크를 하고 있습니다 !

Reply:
커스텀인 경우에는 발생하지않나요?? 이번에 stage에서 생긴 Cannot invoke "java.util.Collection.toArray()" because "c" is null 오류가 이 부분같다고 생각이 드는데.. 지금 로컬에서 커스텀 생성에 class_time값을 null을 주면 npe발생하고있습니다

Reply:
커스텀인 경우 class_time이 null이 되면 안되는 것을 dto에서 어노테이션을 통해 검증을 해야하는데, 정규 강의의 경우 class_time를 null로 추기 때문에 발생하는 문제라고 생각합니다.

Reply:
음.. 그러면 pr에 수정된 코드는 해결된건가요?? 일단 승인하겠습니다!